tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -145,7 +146,8 @@ rm -rf /var/lib/apt/lists/*
 
 if [ "$SYS_LONG_BIT" = "64" ]; then
     if [ "$DEVICE_TYPE" = "Normal" ]; then
-	aria2c -q -d /tmp -o llvm.tar.xz http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-${OS_VERSION}.tar.xz
+    # originally from http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-${OS_VERSION}.tar.xz
+	aria2c -q -d /tmp -o llvm.tar.xz https://onnxruntimetestdata.blob.core.windows.net/models/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-${OS_VERSION}.tar.xz
```

Comment:
>https://onnxruntimetestdata.blob.core.windows.net/models/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-${OS_VERSION}.tar.xz [](start = 34, length = 118)

Just make sure we are not redistributing and if we are then we need to appropriate license file and clearance from CELA

Reply:
instead of hosting it ourselves,
we can try alternatives such as https://github.com/llvm/llvm-project/releases/download/llvmorg-9.0.1/clang+llvm-9.0.1-x86_64-linux-gnu-ubuntu-16.04.tar.xz  ?

Reply:
Yes, and it seems as of today http://releases.llvm.org/9.0.0/clang+llvm-9.0.0-x86_64-linux-gnu-ubuntu-${OS_VERSION}.tar.xz is actually working now.